### PR TITLE
ft/trip-details

### PR DESCRIPTION
**Pull request by blandri**
#### What does this PR do?

 Add read trip details

#### Description of Task to be completed?

- When you click on a trip via details button you should see a page with all the details.

#### How should this be manually tested?

- go on this deployed [preview](https://deploy-preview-24--barefoot-nomad-fe.netlify.app/)
- log in as requester or as manager or as admin

    email:         REQUESTER@gmail.com
   password:   REQUESTER2gmail

 email:         SUPER_ADMIN@gmail.com
   password:   SUPER_ADMIN2gmail

email:         MANAGER1@gmail.com
   password:   MANAGER1gmail

- Click on the details button on trip requests

#### What are the relevant pivotal tracker stories?

none

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by Code-250 — APPROVED** [review]

**Review by Eli250 — APPROVED** [review]

**Comment by ncutixavier** [comment]
You must have updated the PR description & title since you push new changes. Ask @Eli250 or @Code-250 for reference

**Comment by ncutixavier** [comment]
image
You should not fetch users when retrieving a single trip

File: src/routes/index.js [hunk 1]
```diff
@@ -80,7 +81,8 @@ const AllRoutes = (props) => (
           element={<ResetPassword {...props} />}
         />
         <Route exact path="/dashboard/*" element={<DashboardPreview />}>
-          <Route exact path="" element={<Navigate to="trips" />} />
+          <Route path="readone/:id/:role" element={<ReadOneTrip />} />
```
Comment (ncutixavier) [inline src/routes/index.js:84]: Change this route, It doesn't follow a naming convention.  Also, don't pass role in the params

**Review by ncutixavier — CHANGES_REQUESTED** [review]
Remember to change your PR Title to follow naming convention

**Pull request merged**